terraform/environments/core-network-services/locals.tf

```diff
@@ -58,7 +58,11 @@ locals {
     "10.102.0.0/16",
     "10.47.0.0/26",
     "10.47.0.64/26",
-    "10.47.0.128/26"
+    "10.47.0.128/26",
+    "10.40.129.79/32"
```

Comment:
Assuming we only want to check against a single host in devtest, according to the story comments we only need `10.102.1.79/32`, so we don't need to make any changes to this local.